### PR TITLE
Dynamics Translation Service (Certified Publisher)

### PR DESCRIPTION
**Pull request by joshftb**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Screenshot showing 3 operations in successful flow: 
image

Test UI (only for operations that don't need file input):
image

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hello @joshftb,
Thank you very much and congratulations on your new submission. Our Certification Engineers will review it within 1-2 weeks.
Please let us know if you have any questions.
Thank you very much again for working with us.

File: certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,887 @@
...
+            }
+          },
+          {
+            "description": "Select the product version.",
+            "x-ms-summary": "Product Version",
+            "in": "formData",
+            "name": "productVersion",
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "Product_Versions",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json:55]: @joshftb operationId properties should use pascal case for values. Please refer to [coding standards documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).

File: certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,887 @@
...
+          }
+        },
+        "summary": "Download Alignment Result",
+        "operationId": "Align_Download",
+        "parameters": [
+          {
+            "name": "filename",
+            "x-ms-summary": "File name",
+            "description": "Filename returned in alignment output.",
+            "x-example": "0_7e47d31c-f3e3-43ea-b6a8-986cc58861cd_1_BI.fr.label_Aligned.xlf",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json:839]: @joshftb secret-look-a-like-values are not allowed. Please remove them or replace with other string value.

File: certified-connectors/Dynamics 365 Translation Service/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#2266e3",
+    "capabilities": [],
+    "publisher": "Microsoft Corporation"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Dynamics 365 Translation Service/apiProperties.json:47]: @joshftb stackOwner property is missing. Please add it. You can refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json) for more details.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @joshftb,

Please check comments after the review.

Let us know if you have any questions.

File: certified-connectors/Dynamics 365 Translation Service/Readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+## Dynamics 365 Translation Service Connector
```
[the remark this answers is not in the document]
Comment (joshftb) [reply certified-connectors/Dynamics 365 Translation Service/Readme.md]: Done!

File: certified-connectors/Dynamics 365 Translation Service/Readme.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
```
[the remark this answers is not in the document]
Comment (joshftb) [reply certified-connectors/Dynamics 365 Translation Service/Readme.md:2]: Done!

File: certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json (hunk 1 above)
Comment (joshftb) [reply certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json]: Done!

File: certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json (hunk 2 above)
Comment (joshftb) [reply certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json]: Done!

File: certified-connectors/Dynamics 365 Translation Service/apiProperties.json (hunk 3 above)
Comment (joshftb) [reply certified-connectors/Dynamics 365 Translation Service/apiProperties.json]: @v-EgorKozhadei , done with this and the above. Let me know if any further action is required from my side.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @joshftb, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**